buildSrc/build.gradle

```
@@ -123,6 +123,8 @@ dependencies {
   api 'org.jruby.joni:joni:2.2.1'
   api "com.fasterxml.jackson.core:jackson-databind:${props.getProperty('jackson_databind')}"
   api "org.ajoberstar.grgit:grgit-core:5.2.1"
+  api "org.bouncycastle:bc-fips:${props.getProperty('bouncycastle_jce')}"
```

I believe this is optional dependency and should not be included by default (this is build tooling which mostly every single plugin will depend upon:

```suggestion
  if (System.getenv('OPENSEARCH_CRYPTO_STANDARD') == 'FIPS-140-3') {
    api "org.bouncycastle:bc-fips:${props.getProperty('bouncycastle_jce')}"
  }```

---

(we sadly cannot use `BuildParams` here)